bitbots_ipm/launch/ipm.launch

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<launch>
+    <arg name="sim" default="false"/>
+    <arg name="full_image" default="false"/>
+    <arg name="markers" default="false"/>
+    <arg name="rviz" default="false"/>
+
+    <node pkg="ipm_image_node" exec="ipm" name="ipm_line_mask" output="screen">
+        <remap to="/camera/camera_info" from="camera_info"/>
+        <remap to="/line_mask_in_image" from="input"/>
+        <remap to="/line_mask_relative_pc" from="projected_point_cloud"/>
+        <param name="type" value="mask"/>
+        <param name="scale" value="0.2"/>
+        <param name="output_frame" value="base_footprint"/>
+        <param name="use_sim_time" value="$(var sim)"/>
```

Comment:
Could we have a separate .yaml for this?

Reply:
The only parameter that would fit into a config file would be the scale and I think the overhead is not worth it for a single param. The other ones like `type` are more related to the purpose defined in the launch file.